README.md

```diff
@@ -34,7 +34,8 @@ class ActiveRecord::Base
   include Arca::Collector
 end
 
-# load your app
+# load your app. It's important to setup before loading your models because arca
+# works by record callback definitions.
```

Comment:
maybe: "works by wrapping itself around the callback method definitions (`before_save`, `after_save`, etc) and then records how and where those methods are used"